modules/ocf_keyserver/manifests/init.pp

```diff
@@ -0,0 +1,61 @@
+class ocf_keyserver {
+  include ocf::firewall::allow_web
+  include ocf::ssl::default
+  include ocf::packages::docker
+
+  file {
+    '/var/lib/sks':
+      ensure  => directory;
+
+    '/var/lib/sks/sksconf':
+      content => template('ocf_keyserver/sksconf.erb');
+
+    '/var/lib/sks/membership':
+      source  => 'puppet:///modules/ocf_keyserver/membership';
+
+  }
+
+  ocf::systemd::service { 'sks-keyserver':
+    source  => 'puppet:///modules/ocf_keyserver/sks-keyserver.service',
+    require => [
+      Package['docker-ce'],
+      File['/var/lib/sks'],
+    ];
+  }
+
+  class { 'nginx':
+    manage_repo  => false,
+    confd_purge  => true,
+    server_purge => true;
+  }
+
+  Class['ocf::ssl::default'] ~> Class['Nginx::Service']
```

Comment:
it should exist from the `class { 'nginx' }` resource earlier, but this service is near-impossible to clean build from puppet alone anyways because generating the database from a keydump needs to be done by hand. If we can figure out the PV/PVC thing, it may be possible to move this service to Kubernetes, though - we'd just need to make sure that the source IP for recon requests survives all the layers of ingress controllers and whatnot